addons/website/controllers/main.py

```diff
@@ -678,6 +678,8 @@ def get_seo_data(self, res_id, res_model):
             fields.extend(['website_indexed', 'website_id'])
 
         record = request.env[res_model].browse(res_id)
+        if res_model != 'website.page' and 'website.seo.metadata' not in request.env[res_model]._inherit_module.keys():
+            raise UserError(_("You can not optimize SEO for livechat."))
```

Comment:
The message says *for livechat*, but it's only one of possible scenarios, no?

Reply:
> The message says _for livechat_, but it's only one of possible scenarios, no?

@yelizariev , changes has been done.